packages/terra-compact-interactive-list/tests/jest/CompactInteractiveList.test.jsx

```diff
@@ -1087,4 +1087,147 @@ describe('Compact Interactive List', () => {
       expect(mockOnCellSelect).not.toHaveBeenCalled();
     });
   });
+
+  describe('row headers', () => {
+    const cols = [
+      {
+        id: 'Column-0',
+        displayName: 'Col_1',
+        width: '40px',
+        minimumWidth: '20px',
+      },
+      {
+        id: 'Column-1',
+        displayName: 'Col_2',
+        width: '200px',
+        maximumWidth: '300px',
+      },
+      {
+        id: 'Column-2',
+        displayName: 'Col_3',
+        width: '40px',
+      },
+    ];
+
+    const singleColRows = [
+      {
+        id: 'row_1',
+        cells: [{ content: 'Discern Care Set (1)' }],
+      },
+      {
+        id: 'row_2',
+        cells: [{ content: 'Initial observation Care/Day High Severity 99220 (2)' }],
+      },
+      {
+        id: 'row_3',
+        cells: [{ content: 'Arterial Sheath Care (3)' }],
+      },
+      {
+        id: 'row_4',
+        cells: [{ content: 'Sbsq Observation Care/Day High Severity 99226 (4)' }],
+      },
+      {
+        id: 'row_5',
+        cells: [{ content: 'Arterial Sheath Care (5)' }],
+      },
+    ];
+
+    const singleCol = [
+      {
+        id: 'Column-0',
+        displayName: 'Col_1',
+        width: '40px',
+      },
+    ];
+
+    it('list with one column should match a snapshot', () => {
+      const wrapper = enzymeIntl.mountWithIntl(
+        <CompactInteractiveList
+          id="compact-interactive-list-with-one-column"
+          rows={singleColRows}
+          columns={singleCol}
+        />,
+      );
+      expect(wrapper).toMatchSnapshot();
```

Comment:
Nit: IIRC we are kind of moving away from doing these kinds of snapshot tests? These tend to be hard to verify and we just overwrite the results anyways.

Reply:
I added the snapshot to ensure the component rendered with correct role, but jest test checks the same so I am willing to remove it. It is a bit extra.

Reply:
Removed in https://github.com/cerner/terra-framework/pull/2110/commits/76ce35a6f621e028bcd7d31645e48727199ca632